### ISSUE TITLE
Create MD to MDX mini-transition guide

### ISSUE THREAD

**Issue by sarah11918** [withastro/docs#1122]
Additional content for migration guide.

Not just "enable legacy flag until ready" but some concrete "here's how to change your files over."

**Comment by dkotik** [comment]
This is sorely needed! One major issue I ran into is that Astro.glob("content/*.mdx") returns an object with `url` property `undefined`. It is impossible to use it together with getStaticPaths without passing a slug in the frontmatter. Strangely, while regular markdown globs include `file` which is used in the documentation for getStaticPaths, MDX does not have the same property and `url` is `undefined`.

**Comment by dkotik** [comment]
Ok, I was able to solve my issue: glob MUST load files from "pages" folder; otherwise returned .url is undefined. It wasn't an issue for regular .md files. So glob path matters a whole lot. This needs to be documented.

**Comment by kevinzunigacuellar** [comment]
I would like to pin this [discord thread](https://discord.com/channels/830184174198718474/1001929767756496906) for anyone that needs more information or help migrating from `.md` to `.mdx`

**Comment by sarah11918** [comment]
I'm going to close this issue because we did get some docs content up in the Migration guide:  https://docs.astro.build/en/migrate/#converting-existing-md-files-to-mdx

@dkotik I'm going to speak to engineering about items needing to be in `src/pages` to return `url`. This doesn't immediately surprise me because the file only makes a page route if it's in `/pages/`, so it might not otherwise have a `url` associated with it? But if it's a change from `.md` then it's worth bringing up to them.

I know there is more work going on as we speak with the intention of bringing parity to `.mdx` / `.md` usage. So even the guidance we gave on Thursday has changed (still works, but no longer necessary!) as we bring the MDX experience closer to what we had with Astro MD.
